instance-scheduler/main.go

```diff
@@ -143,6 +170,12 @@ type InstanceCount struct {
 	skippedAutoScaled int
 }
 
+type RDSInstanceCount struct {
+	RDSActedUpon         int
+	RDSSkipped           int
+	RDSSkippedAutoScaled int
```

Comment:
your struct type has `RDSSkippedAutoScaled` field, but you are not using it later in code

Reply:
I noticed you also test it in unit tests, but in your main code logic, you never touch it.
If this is not needed, it should be removed.

Reply:
removed